.idea/runConfigurations/arx_3_7_0_osx_64_main.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="arx-3.7.0-osx-64-main" type="JarApplication" factoryName="JAR Application">
```

Comment:
This configuration points to a specific build of ARX. I thinkt that it should be removed. If not, please explain why it is needed.